Sources/Overlays/_Testing_Foundation/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ add_library(_Testing_Foundation
   ReexportTesting.swift)
 
 target_link_libraries(_Testing_Foundation PUBLIC
+  _TestingInternals
```

Comment:
The Foundation overlay shouldn't need to link to this module.

Reply:
I found that it did - there was a missing module error otherwise.

Reply:
The overlay doesn't import it and it's a static library, so if it's complaining, that probably aligns with what @stmontgomery noted in his comment.